build.sbt

```diff
@@ -0,0 +1,25 @@
+import sbt.Keys._
+import sbt._
+import uk.gov.hmrc.versioning.SbtGitVersioning
+
+val appName = "json-encryption"
+
+val dependencies = Seq(
+  "com.typesafe.play" %% "play-json"   % "2.6.5" % "provided",
```

Comment:
I'm a bit worried by this play 2.6.5 even though it has nothing to do with upgrading crypto

Reply:
Changed to from hardcoded 2.6.5 to PlayVersion.current.